### DISCUSSION TITLE
Andres Gonzalez - Predicting Football Match Outcomes Using Bayesian Poisson-Gamma Distribution

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-web#1268] [General]
# Andres Gonzalez - Predicting Football Match Outcomes Using Bayesian Poisson-Gamma Distribution

https://okcomputer626.quarto.pub/andres-gonzalez/posts/MLS%20Predictions/Predicting%20Football%20Match%20Outcomes%20Using%20Bayesian%20Poisson-Gamma%20Distribution.html

**Comment by cderv** [comment]
@OKcomputer626 Unfortunately, wrong config: Reported upstream already 
- https://github.com/OKcomputer626/Projects-and-Stuff/issues/1
